airbyte-integrations/connectors/source-hubspot/acceptance-test-config.yml

```diff
@@ -5,21 +5,57 @@ tests:
   connection:
     - config_path: "secrets/config.json"
       status: "succeed"
+    - config_path: "secrets/config_oauth.json"
+      status: "succeed"
     - config_path: "integration_tests/invalid_config.json"
       status: "failed"
   discovery:
     - config_path: "secrets/config.json"
   basic_read:
     - config_path: "secrets/config.json"
       # TODO: permissions error with Workflows stream for Test Account
-      configured_catalog_path: "sample_files/configured_catalog_without_workflows.json"
-#  incremental: fixme (eugene): '<=' not supported between instances of 'int' and 'str'
-#    - config_path: "secrets/config.json"
-#      configured_catalog_path: "sample_files/configured_catalog.json"
-#      future_state_path: "integration_tests/abnormal_state.json"
-#      cursor_paths:
-#        subscription_changes: ["timestamp"]
-#        email_events: ["timestamp"]
+      configured_catalog_path: "sample_files/configured_catalog.json"
+      empty_streams:
+        [
+          "contact_lists",
+          "campaigns",
+          "tickets",
+          "subscription_changes",
+          "quotes",
+          "email_events",
+          "engagements",
+          "forms",
+          "products",
+          "workflows",
+        ]
+    - config_path: "secrets/config_oauth.json"
+      # TODO: permissions error with Workflows stream for Test Account
+      configured_catalog_path: "sample_files/configured_catalog.json"
+      empty_streams:
+        [
+          "companies",
+          "deals",
+          "owners",
+          "contact_lists",
+          "campaigns",
+          "tickets",
+          "subscription_changes",
+          "quotes",
+          "email_events",
+          "engagements",
+          "forms",
+          "products",
+          "workflows",
+        ]
+  #  incremental: fixme (eugene): '<=' not supported between instances of 'int' and 'str'
```

Comment:
Same here, do we have an issue for this? If yes, then, maybe, we should put issue number here too? Or if we don't have an issue then create one and add issue number.

Reply:
https://github.com/airbytehq/airbyte/issues/6509